src/mongo/platform/random.cpp

```diff
@@ -103,7 +103,7 @@ SecureRandom* SecureRandom::create() {
     return new WinSecureRandom();
 }
 
-#elif defined(__linux__) || defined(__sunos__) || defined(__APPLE__) || defined(__freebsd__)
+#elif defined(__linux__) || defined(__sunos__) || defined(__APPLE__) || defined(__freebsd__) || defined(__FreeBSD__) || defined(__FreeBSD_kernel__) || defined(__gnu_hurd__)
```

Comment:
Are all three of _freebsd__ __FreeBSD__ and __FreeBSD_kernel__ required? What does each of them represent?